### DISCUSSION TITLE
[postcss-preset-env] now has built-in Typescript declarations

### DISCUSSION THREAD

**Discussion by romainmenke** [DefinitelyTyped/DefinitelyTyped#64057] [Issues with a @types package] [answered]
https://www.npmjs.com/package/postcss-preset-env

From now on we will publish `postcss-preset-env` with Typescript declarations.

I have no idea how `DefinitelyTyped` handles cases like this:
- leave everything as-is
- remove the library from this repo
- ...

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
Types on DefinitelyTyped are normally removed once the original library gets type definitions. I've opened #64058 to remove the types for postcss-preset-env

**Reply by MysteryBlokHed** [reply]
PR just got merged 😄

**Reply by romainmenke** [reply]
Thank you for the super fast response and updates here 🎉
